pom.xml

```diff
@@ -64,23 +64,25 @@
         <!-- Need to explicitly set this otherwise it will be overridden from common-docker pom. -->
         <docker.file>Dockerfile.${docker.os_type}</docker.file>
         <docker.tag>${project.version}-${docker.os_type}</docker.tag>
+        <io.confluent.schema-registry.version>${confluent.version.range}</io.confluent.schema-registry.version>
+        <io.confluent.blueway.version>${confluent.version.range}</io.confluent.blueway.version>
```

Comment:
@cadonna That property is defined in the common repo pom file. The value for this branch is:
`[6.1.0-0, 6.1.1-0)`

Which means it will the latest/greatest 6.1.0-* artifact.